Tester.java

```diff
@@ -429,9 +292,10 @@ private void doTest() {
                 Credits: 16
                 Journey Points: 0
                 ********************
-                Actual:"""+"\n"+Wayward.viewAPass(cId4));
+                Actual:
+                """ + Wayward.viewAPass(cId4));
```

Comment:
Just alot of the same comments. The formatting is off, make sure it's all spaced nicely and easy to read.

Also simplify some stuff as you have extra variables when you don't need to.

And option 1 and 2 are missing.